src/components/MessageList/MessageList.tsx

```diff
@@ -38,6 +38,7 @@ import type {
   DefaultReactionType,
   DefaultUserType,
 } from '../../types/types';
+import type { GroupStyle } from './utils';
```

Comment:
Let's move this import to line 28, above the MessageProp type and with a line in between.